### DISCUSSION TITLE
Show git tag instead of hash

### DISCUSSION THREAD

**Discussion by palango** [zimfw/zimfw#510] [Q&A] [answered]
I'm coming from oh-my-zsh and it used to display the tag when one was checked out instead of the commit hash:
![Screenshot 2023-05-05 at 11 26 13](https://user-images.githubusercontent.com/90851/236422669-9dea5e72-e295-4476-a6d5-c603d78571ad.png)

With the default prompt, I just see the commit hash, instead of the tag name. Is that something that is supported?

![Screenshot 2023-05-05 at 11 26 51](https://user-images.githubusercontent.com/90851/236422695-7c614e26-372f-45bf-af12-613b04aaf126.png)

**Comment by ericbn** [comment, marked as answer]
Hi @palango. Our asciitheme is inspired by [starship](https://starship.rs/), and as far as I've used that theme, it also does not show the tag name by default, so that's why we kept our theme that way.

You can customize asciiship to show the commits from nearest tag count (when in 'detached HEAD' state) using:
```zsh
  zstyle ':zim:git-info:position' format 'HEAD %F{green}(%p)'
  zstyle ':zim:git-info:keys' format \
      'status' '%S%I%i%A%B' \
      'prompt' ' on %%B%F{magenta}%b%p%s${(e)git_info[status]:+" %F{red}[${(e)git_info[status]}]"}%f%%b'
```

Our themes use our [git-info](https://github.com/zimfw/git-info) module to show git information. You can learn more about how to customize it by checking its [settings](https://github.com/zimfw/git-info/blob/master/README.md#settings) documentation.

**Reply by palango** [reply]
Thanks a lot!
